saltyrtc/server/bin.py

```diff
@@ -142,9 +143,10 @@ def generate(key_file):
 @click.option('-sk', '--sslkey', type=click.Path(exists=True), help=_h("""
 Path to a file that contains the SSL private key. Will be read from
 the SSL certificate file if not present."""))
-@click.option('-k', '--key', type=click.Path(), help=_h("""
+@click.option('-k', '--key', type=click.Path(), multiple=True, help=_h("""
```

Comment:
Tested it manually beforehand. Maybe we should add a test for that... just in case.